app/controllers/admin/merchant_accounts_controller.rb

```diff
@@ -4,28 +4,6 @@ class Admin::MerchantAccountsController < Admin::BaseController
   def show
     @merchant_account = MerchantAccount.find_by(id: params[:id]) || MerchantAccount.find_by(charge_processor_merchant_id: params[:id]) || e404
     @title = "Merchant Account #{@merchant_account.id}"
-    load_live_attributes
+    render inertia: "Admin/MerchantAccounts/Show", props: { merchant_account: Admin::MerchantAccountPresenter.new(merchant_account: @merchant_account).props }
   end
-
-  private
-    def load_live_attributes
```

Comment:
Should we keep this around for now to support `admin_old`?

Reply:
It's no longer compatible after changing it to an array, but I don't think it's worth adding when we're this close to deleting the old code anyway (the old page does render, just without live attributes)